common/source/docs/common-wiki-editing-setup.rst

```diff
@@ -82,12 +82,19 @@ The main steps for building the docs are:
    .. code-block:: bash
 
        cd /vagrant
+       source env/ardupilot_wiki_env/bin/activate
```

Comment:
Let's get this into the .bashrc  on the VM so we don't need to worry about it any more on the VM.

Reply:
In fact, both lines should be in the .bashrc